modules/storage-persistent-storage-pv.adoc

```diff
@@ -45,7 +45,7 @@ once it is released.
 - HostPath
 - iSCSI
 - NFS
-// - Azure Disk
+- Azure Disk
```

Comment:
It looked like this list was previously alphabetical. Should Azure Disk go below AWS?

Reply:
You're right - I made this alphabetical in 4.1 from 3.11, and moved this line.